### PR TITLE
patch bcoin.keyring typo

### PR DESCRIPTION
**Pull request by ryan-lingle**
resolves bcoin-org/bcoin#801

### CODE REVIEW

File: guides-markdown/crowdfund-tx.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ In most Bitcoin transactions, when a transaction is signed, the entirety of the
 The `ALL|ANYONECANPAY` flag indicates that a signature is committing all of the outputs and just one input. The suggested use case of this proposed in *Mastering Bitcoin* is a kickstarter-like crowdfunding application. Consider the situation where you have a fundraising goal, say 1BTC. You'd like multiple people to contribute to your goal and you want to prove to them that they won't have to pay *unless* you reach your goal (i.e. the transaction is invalid and won't be accepted by the network if attempted to be sent). This means you'd be committing to one output, the one that sends the funds to the charity or project you'd like to donate to, and only one input, your contribution. This allows multiple users to contribute to the same transaction and that transaction won't be valid until it's fully funded.
 
 Here's how it's explained in *Mastering Bitcoin*:
-> ALL|ANYONECANPAY  
+> ALL|ANYONECANPAY
```
Comment (pinheadmz) [inline guides-markdown/crowdfund-tx.md]: Ooh nice linting as well! bonus.

Comment (pinheadmz) [reply guides-markdown/crowdfund-tx.md]: Although, we usually prefer commits to be more focused so in the future the extra-space-removal would ideally be a separate PR, tedious though it is.

**Comment by pinheadmz** [comment]
LGTM! Thank you.

File: guides-markdown/crowdfund-tx.md (hunk 1 above)
Comment (ryan-lingle) [reply guides-markdown/crowdfund-tx.md]: Oops, that was not intentional! I must have a javascript linter installed on my text editor that I am unaware of.

**Pull request merged**